extension/src/popup/components/identicons/KeyIdenticon/index.tsx

```diff
@@ -15,41 +15,76 @@ interface IdenticonWrapperElProps {
 }
 
 interface KeyIdenticonProps extends IdenticonWrapperElProps {
+  allowCopy?: boolean;
```

Comment:
NIT: we should follow boolean naming convention with `isCopyAllowed` (or something similar)

Reply:
Done in 4b135e5b899b32ec2040d54ce0438b559fe3b135